canonical-web-and-design/jaas.ai
# Use @canonical/latest-news for blog feed

## Done

Use @canonical/latest-news for blog feed

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029/community
- Verify blog feed loads
